snuba/web/views.py

```diff
@@ -274,6 +275,12 @@ def _trace_transaction(dataset: Dataset) -> None:
             scope.span.set_tag("dataset", get_dataset_name(dataset))
             scope.span.set_tag("referrer", http_request.referrer)
 
+    transaction = Hub.current.scope.transaction
+    if transaction is not None and settings.UPDATE_TRANSACTION_NAME:
+        transaction.name = (
+            f"{transaction.name}__{get_dataset_name(dataset)}__{http_request.referrer}"
+        )
+
```

Comment:
Can you just do `scope.transaction = "new name"` inside the configure_scope block?

Reply:
Yep, good call.